enumerable.rb

```diff
@@ -0,0 +1,110 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+
+module Enumerable
+  def my_each
+    return to_enum unless block_given
+
+    arr.length.times do |n|
+      yield arr[n]
+    end
+  end
+
+  def my_each_with_index
+    return to_enum unless block_given
+
+    arr.length.times do |n, i|
+      yield(arr[i], n)
+    end
+  end
+
+  def my_select
+    return to_enum unless block_given
+
+    list = []
+
+    arr.my_each | filter |
+      if filter != 'accepted'
+        list.push(filter)
+        yield filter
+      end
+  end
```

Comment:
- [ ] enumerable #my_select has three errors. Please have a look at the attachment and remove those errors. 

<img width="679" alt="3" src="https://user-images.githubusercontent.com/58504655/103562876-a4874700-4edd-11eb-9963-e2c3042545c7.PNG">